src/bin/cratesfyi.rs

```diff
@@ -52,6 +52,10 @@ enum CommandLine {
     StartWebServer {
         #[structopt(name = "SOCKET_ADDR", default_value = "0.0.0.0:3000")]
         socket_addr: String,
+
+        /// Reload templates when they're changed
+        #[structopt(long = "reload")]
```

Comment:
Nit: maybe let's rename the flag to `--reload-templates`, otherwise it might seem it reloads the whole application.